include/wx/clntdata.h

```diff
@@ -158,5 +159,38 @@ class WXDLLIMPEXP_BASE wxClientDataContainer
 
 };
 
+// This class is a replacement for wxClientDataContainer, and unlike
+// wxClientDataContainer the wxSharedClientDataContainer client data is
+// possible to copy (as a shared ptr) when instances of it are cloned.
```

Comment:
We can't really say "is possible to copy", maybe something like this would be more clear:
```suggestion
// copyable, so it can be copied when objects containing it are cloned.
```